README.md

```diff
@@ -1,43 +1,197 @@
+        </tr>
+    </tbody>
+</table>
 
 ## License
 OpenVINO™ Toolkit is licensed under [Apache License Version 2.0](LICENSE).
 By contributing to the project, you agree to the license and copyright terms therein and release your contribution under these terms.
 
-## Resources
-* Docs: https://docs.openvino.ai/
-* Wiki: https://github.com/openvinotoolkit/openvino/wiki
-* Issue tracking: https://github.com/openvinotoolkit/openvino/issues
-* Storage: https://storage.openvinotoolkit.org/
-* Additional OpenVINO™ toolkit modules: https://github.com/openvinotoolkit/openvino_contrib
-* [Intel® Distribution of OpenVINO™ toolkit Product Page](https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit.html)
-* [Intel® Distribution of OpenVINO™ toolkit Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNotes)
+## Documentation
+
+### User documentation
+
+The latest documentation for OpenVINO™ Toolkit is availabe [here](https://docs.openvino.ai/). This documentation contains detailed information about all OpenVINO components and provides all important information which could be needed if you create an application which is based on binaty OpenVINO distribution or own OpenVINO version without source code modification.
+
+### Developer documentation
+
+[Developer documentation](#todo-add) contains information about architectural decisions which are applied inside the OpenVINO components. This documentation has all necessary information which could be needed in order to contribute to OpenVINO.
+
+## Tutorials
+
+The list of OpenVINO tutorials:
+
+- [Jupiter notebooks](https://github.com/openvinotoolkit/openvino_notebooks)
+
+## Products which use OpenVINO
+
+- [OpenCV](https://opencv.org/)
+- [ONNX Runtime](https://onnxruntime.ai/)
+- [OpenVINO™ Integration with TensorFlow](https://www.intel.com/content/www/us/en/developer/tools/devcloud/edge/build/ovtfoverview.html)
+
+## System requirements
+
+The full information about system requirements depends on platform and available in section `System requirement` on dedicated pages:
+- [Linux](https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_linux.html)
+- [Windows](https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_windows.html)
+- [macOS](https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_macos.html)
+- [Raspbian](https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_raspbian.html)
+
+## How to build
+
+Please take a look to [OpenVINO Wiki](https://github.com/openvinotoolkit/openvino/wiki#how-to-build) to get more information about OpenVINO build process.
+
+## How to contribute
+
+See [CONTRIBUTING](./CONTRIBUTING.md) for details. Thank you!
+
+## Get a support
 
-## Support
 Please report questions, issues and suggestions using:
 
 * The [`openvino`](https://stackoverflow.com/questions/tagged/openvino) tag on StackOverflow\*
```

Comment:
should we rearrange ? E.i. recommend github issues first

Reply:
Fixed